entries/r/readme.com.json

```
@@ -1,7 +1,6 @@
 {
   "ReadMe": {
-    "domain": "readme.io",
-    "img": "readme.io.png",
+    "domain": "readme.com",
```

```suggestion
    "domain": "readme.com",
    "additional-domains": [
      "readme.io"
    ],
```
An `additional-domains` entry might help password managers (e.g. for browser auto-fill support).